### PR TITLE
Tests and minor controller fix

### PR DESCRIPTION
**Pull request by MateuszUrbanowicz**

### CODE REVIEW

**Comment by AdrianSkierniewski** [comment]
We should check some invalid cases too

File: tests/functional/AdminOptionsCest.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            ]
+        );
+    }
+
+    public function updateOptionValue(FunctionalTester $I)
+    {
+        $I->wantTo('update value of option as admin user');
+        $I->loginAsAdmin();
+
+        $I->sendPATCH(
```
Comment (AleksandraAugustyn) [inline tests/functional/AdminOptionsCest.php]: Why we are sending PATCH? We are using PUT in update methods.

**Comment by AleksandraAugustyn** [comment]
Invalid cases shoudl be:

* Trying to get not existing category
* Trying to get not existing option key
* Trying to get not existing option key lang value
* Trying to update not existing category
* Trying to update not existing option key
* Trying to update not existing option key lang value

**Comment by AdrianSkierniewski** [comment]
:+1:

**Comment by AleksandraAugustyn** [comment]
:+1:

**Pull request merged**